From e3c65ebb4a723154a206717d2e83666a439f3783 Mon Sep 17 00:00:00 2001
From: Zhenbo Gao <zhenbo.gao@windriver.com>
Date: Tue, 21 Aug 2018 06:16:28 +0000
Subject: [PATCH 003/176] Revert "tipc: use only positive error codes in
 messages"

This reverts commit ef4721707447b77e741191cb9f2d0a18be93d79a.

A big patch set for upgrading TIPC(to v4.18) will come soon

Signed-off-by: Zhenbo Gao <zhenbo.gao@windriver.com>
---
 net/tipc/msg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/tipc/msg.c b/net/tipc/msg.c
index ee5adf0..ab30876 100644
--- a/net/tipc/msg.c
+++ b/net/tipc/msg.c
@@ -547,7 +547,7 @@ bool tipc_msg_lookup_dest(struct net *net, struct sk_buff *skb, int *err)
 		return false;
 	if (msg_errcode(msg))
 		return false;
-	*err = TIPC_ERR_NO_NAME;
+	*err = -TIPC_ERR_NO_NAME;
 	if (skb_linearize(skb))
 		return false;
 	msg = buf_msg(skb);
-- 
1.7.9.5

